config.schema.json

```diff
@@ -806,7 +857,6 @@
                 }
             },
             "required": [
-                "ad",
```

Comment:
Please keep ad in required and add the use_existing_ad property as required as well